tests/envs/test_registration.py

```diff
@@ -24,11 +26,30 @@ def __init__(self, arg1, arg2, arg3):
 
 
 def test_make():
-    env = envs.make("CartPole-v0")
-    assert env.spec.id == "CartPole-v0"
+    env = envs.make("CartPole-v1")
+    assert env.spec.id == "CartPole-v1"
     assert isinstance(env.unwrapped, cartpole.CartPoleEnv)
 
 
+@pytest.mark.parametrize(
```

Comment:
We should probably test some edge case environment IDs:
```
# Valid
MyAwesomeEnv-vfinal-v0
MyAwesomeEnv-vfinal
MyAwesomeEnv--
MyAwesomeEnv-v
```
These edge cases should all be valid.

Reply:
Speaking of edge cases, would you consider adding a dev dependency? [Hypothesis](https://pypi.org/project/hypothesis/) is very nice to generate all the edge cases you didn't think of and to catch hidden bugs. For example we could use the [hypothesis.strategies.from_regex()](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.from_regex) strategy and use the same regex as in the code to automatically generate tests.

Reply:
CC @jkterry1

Reply:
ya thats fine for tests for me